.changelog/4733.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:feature
+core/cli: Add project templating cli
```

Comment:
We should probably be a bit more explicit here before merging this in, since this is something users will be reading and will be under the *features* heading. Also, this is an HCP only feature so we should mention that too! Maybe a good example to get inspiration from is our 0.10.0 release notes: https://github.com/hashicorp/waypoint/releases/tag/v0.10.0

Reply:
Actually, this seems to be a beta feature, not ready for full usage, so let's remove the changelog for now